xrpl/models/ledger_objects/account_root.py

```diff
@@ -0,0 +1,56 @@
+"""Models for the Ledger Object `AccountRoot`"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import Optional, Union
+
+from xrpl.models.ledger_objects.ledger_entry_type import LedgerEntryType
+from xrpl.models.ledger_objects.ledger_object import LedgerObject
+from xrpl.models.required import REQUIRED
+from xrpl.models.utils import require_kwargs_on_init
+
+
+@require_kwargs_on_init
+@dataclass(frozen=True)
+class AccountRoot(LedgerObject):
```

Comment:
Please add the docstrings from xrpl.org to each of the fields (for all the models).